tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Requests/GetOrderItemRequest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Responses;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Utils;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Requests
+{
+    internal sealed class GetOrderItemRequest
+    {
+        private readonly Request _request;
+        private string _getOrderItemUrl;
+
+        public GetOrderItemRequest(Request request,
+            string orderingApiBaseAddress,
+            string orderId,
+            string catalogueItemType)
+        {
+            _request = request;
+            OrderId = orderId;
+            CatalogueItemType = catalogueItemType;
+
+            _getOrderItemUrl = $"{orderingApiBaseAddress}/api/v1/orders/{orderId}/order-items";
```

Comment:
The appending of the catalogue item type could be done here rather than in the `ExecuteAsync` method as it cannot change.